receiver/statsdreceiver/internal/protocol/statsd_parser.go

```diff
@@ -430,6 +431,28 @@ func parseMessageToMetric(line string, enableMetricType bool, enableSimpleTags b
 
 				kvs = append(kvs, attribute.String(k, v))
 			}
+		case strings.HasPrefix(part, "c:"):
+			// As per DogStatD protocol v1.2:
+			// https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12
+			containerID := strings.TrimPrefix(part, "c:")
+
+			if containerID != "" {
+				kvs = append(kvs, attribute.String("container.id", containerID))
```

Comment:
Please use https://github.com/open-telemetry/opentelemetry-collector/blob/main/semconv/v1.22.0/generated_resource.go#L431 instead of hardcoded name

Reply:
Nice! Didn't know about this 🙂 
Just changed it